### PR TITLE
Add English back to Language list

### PR DESCRIPTION
**Pull request by david-allison-1**
## Goals

Please check the tests to see if you're happy with breakages if the language is changed

## Purpose / Description
I noticed that English was no longer a selectable language

## Approach
Adds English back in

## How Has This Been Tested?
On my device, and with regression tests.

## Learning (optional, can help others)

English is a language

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code

### CODE REVIEW

**Comment by timrae** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java  2
         

Clones added
============
- AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java  1
- AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5572324&bid=18270302)

File: AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.java [hunk 1]
```diff
@@ -52,7 +57,8 @@ public static Locale getLocale() {
      * @param localeCode The locale code of the language
      * @return The {@link Locale} for the given code
      */
-    public static Locale getLocale(String localeCode) {
+    @NonNull
+    public static Locale getLocale(@Nullable String localeCode) {
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.java:61]: Codacy Issue found: [Avoid reassigning parameters such as 'localeCode'](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5572324)

File: AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "zh_CN", //zh-CN
+            "zh_TW"  //zh-TW
+    );
+
+    @Test
+    public void testNoLanguageIsRemoved() {
+        HashSet<String> languages = new HashSet<>();
+        Collections.addAll(languages, LanguageUtil.APP_LANGUAGES);
+
+        List<String> previousLanguages = new ArrayList<>(Arrays.asList(PREVIOUS_LANGUAGES));
```
Comment (timrae) [inline AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java:61]: Codacy Issue found: [Unnecessary use of fully qualified name 'Arrays.asList' due to existing static import 'java.util.Arrays.asList'](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5572324)

File: AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    private static final String[] CURRENT_LANGUAGES = {"af", "am", "ar", "az", "be", "bg", "bn", "ca", "ckb", "cs", "da",
+            "de", "el", "en", "eo", "es-AR", "es-ES", "et", "eu", "fa", "fi", "fil", "fr", "fy-NL", "ga-IE", "gl", "got",
+            "gu-IN", "he", "hi", "hr", "hu", "hy-AM", "id", "is", "it", "ja", "jv", "ka", "kk", "km", "ko", "ku",
+            "ky", "lt", "lv", "mk", "mn", "mr", "ms", "my", "nl", "nn-NO", "no", "pa-IN", "pl", "pt-BR", "pt-PT",
+            "ro", "ru", "sk", "sl", "sq", "sr", "ss", "sv-SE", "sw", "ta", "te", "tg", "th", "ti", "tl", "tn", "tr",
+            "ts", "tt-RU", "uk", "ur-PK", "uz", "ve", "vi", "wo", "xh", "yu", "zh-CN", "zh-TW", "zu" };
+
+    /** Languages which were removed for good reason */
+    private static final HashSet<String> previousLanguageExclusions = Sets.newHashSet(
+            "pt_PT", //pt-PT
```
Comment (mikehardy) [inline AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java:49]: Mmm - actually I think this needs to be verified, we may need to transpose xx-yy to xx_yy to go from crowdin to java - I'll take that as a todo personally

**Comment by timrae** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java  2
         

Clones added
============
- AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsLanguageRegressionTest.java  1
- AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5572324&bid=18270302)

File: AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.java (hunk 1 above)
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.java:61]: Codacy Issue found: [Avoid reassigning parameters such as 'localeCode'](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5572324)

**Pull request merged**